Documentation/Usage/MobileCore.md

```diff
@@ -378,7 +408,7 @@ MobileCore.getSdkIdentities { (ids, error) in
 ###### Objective-C
 
 ```objective-c
-[AEPMobileCore getSdkIdentities:^(NSString * _Nullable ids, enum AEPError error) {
+[AEPMobileCore getSdkIdentities:^(NSString * _Nullable ids, NSError* _Nullable error) {
```

Comment:
There are several places in the Identity's docs stilling use `AEPError`, can you update them as well?